tip-wlan/resources/environments/dev-amazon-tip.yaml

```diff
@@ -72,76 +60,33 @@ opensync-gw-cloud:
   persistence:
```

Comment:
ovsdb and mqtt host should also be reverted back to use demo.lab.wlan.tip.build.